src/fieri/app/models/license_worker.rb

```diff
@@ -0,0 +1,43 @@
+require 'sidekiq'
+require 'net/http'
+
+class LicenseWorker
+  include ::Sidekiq::Worker
+
+  # Acceptable licenses were determined and documented in
+  # https://github.com/chef-cookbooks/cookbook-quality-metrics/blob/master/quality-metrics/qm-003-license.md
+  ACCEPTABLE_LICENSES = ['Apache 2.0', 'MIT', 'GNU Public License 2.0', 'GNU Public License 3.0'].freeze
```

Comment:
I suspect we're going to see a number of variations on these strings. I wonder if we should consider updating QM003 to use [the SPDX list of license identifiers](https://spdx.org/licenses/) like Habitat's metadata does. 

@nathenharvey Do you have any opinions about which way to encourage cookbook authors to go for consistent license string formatting?